src/Evolution/Systems/NewtonianEuler/Characteristics.hpp

```diff
@@ -70,6 +70,8 @@ struct CharacteristicSpeedsCompute : CharacteristicSpeeds<Dim>, db::ComputeTag {
       tmpl::list<Velocity<DataVector, Dim>, SoundSpeed<DataVector>,
                  ::Tags::Normalized<::Tags::UnnormalizedFaceNormal<Dim>>>;
 
+  using volume_tags = tmpl::list<>;
```

Comment:
this shouldn't be necessary as it is the default.

Reply:
Removed!